resources/generate_book.py

```diff
@@ -560,6 +560,14 @@ def make_book(rm: ResourceManager, i18n: I18n, local_instance: bool = False):
             text('The $(thing)Thatch Bed$() is a primitive bed which can be used to set your spawn, although not to sleep through the night. To make a thatch bed, place two $(thing)Thatch$() blocks adjacent to each other. Then, right click with a $(thing)Large Raw Hide$(). Large hides are dropped by larger animals, like $(thing)bears$() and $(thing)cows$().'),
             multiblock('Thatch Bed', 'A constructed thatch bed.', False, mapping={'0': 'tfc:thatch_bed[part=head,facing=west]', 'D': 'tfc:thatch_bed[part=foot,facing=east]'}, pattern=((' D ', ' 0 '),)),
         )),
+        entry('size_and_weight', 'Size and Weight', 'tfc:wood/chest/kapok', pages=(
+            text('Every item has a $(thing)Size ⇲$() and $(thing)Weight \u2696$(). An item\'s size and wieght is shown on the item\'s $(thing)tooltip$(), which appears when you hover over it with your mouse.'),
+            text('Size determines what storage blocks an item can fit inside of.$(br)$(li)$(thing)Tiny$() items fit in anything.$(li)$(thing)Very Small$() items fit in anything.$(li)$(thing)Small$() items are the largest item that will fit in $(l:mechanics/decay#small_vessels)Small Vessels$()$().$(li)Normal$() items are the largest that will fit in $(l:mechanics/decay#small_vessels)Large Vessels$().', title='Size ⇲'),
+            text('$(li)$(thing)Large$() items are the largest that will fit in Chests. $(l:getting_started/pit_kiln)Pit Kilns$() or Placed Items can hold four.$(li)$(thing)Very Large$() items are stored alone in Pit Kilns and Placed Items.$(li)$(thing)Huge$() items do not fit in any normal storage device. They count towards overburdening.'),
+            text('$(thing)Overburdening happens when you carry $(thing)Huge$() items. Carrying just one Huge item causes you to become exhausted, making your food burn quicker. Carrying two or more gives you the $(thing)Overburdened$() status effect, which makes movement very slow.$(br)Devices that hold their contents when dropped, such as $(l:mechanics/barrels)Barrels$() and $(l:mechanics/crucible)Crucibles$(), as well as $(l:mechanics/anvils)Anvils$() are Huge. $(l:mechanics/animal_husbandry#horses)Horses$() can be overburdened.', title='Overburdening'),
```

Comment:
It shouldn't just be huge - it should be huge + very heavy. Barrels are huge, but their weight changes when sealed vs. not sealed, which triggers the overburdening effect. (And if that isn't how it's implemented, it should be made that way).